modules/auxiliary/admin/kerberos/inspect_ticket.rb

```diff
@@ -0,0 +1,62 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::Kerberos::Client
+  include Msf::Exploit::Remote::Kerberos::Ticket
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Kerberos Ticket Inspecting',
+        'Description' => %q{
+          This module outputs the contents of a ccache/kirbi file
+        },
+        'Author' => [
+          'Dean Welch' # Metasploit Module
+        ],
+        'References' => [],
+        'License' => MSF_LICENSE,
+        'Notes' => {
+          'Stability' => [],
+          'SideEffects' => [],
+          'Reliability' => [],
+          'AKA' => ['Klist']
+        }
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('NTHASH', [ false, 'The krbtgt/service nthash' ]),
+        OptString.new('AES_KEY', [ false, 'The krbtgt/service AES key' ]),
```

Comment:
I think we'll want similar key validation logic for these 👀

Reply:
yup absolutely